README.md

```diff
@@ -102,7 +104,7 @@ Or on CodeSandBox at the following url: <a href="https://codesandbox.io/s/04wxlo
 
 A set of eight render props for rendering controls in different positions around the carousel.
 
-- Valid render props for the eight positions are `renderTopLeftControls`, `renderTopCenterControls`, `renderTopRightControls`, `renderCenterLeftControls`, `renderCenterCenterControls`, `renderCenterRightControls`, `renderBottomLeftControls`, `renderBottomCenterControls`, and `renderBottomRightControls`.
+- Valid render props for the eight positions are `renderTopLeftControls`, `renderTopCenterControls`, `renderTopRightControls`, `renderCenterLeftControls`, `renderCenterCenterControls`, `renderCenterRightControls`, `renderBottomLeftControls`, `renderBottomCenterControls`, and `renderBottomRightControls`. The default props are set as `renderCenterLeftControls` for `Previous` button and `renderCenterRightControls` for the `Next` button. To change the position, the default positions need to be disabled by setting them to null.  
```

Comment:
I would make you addition a separate bullet point so it doesn't get lost in the list of props

Reply:
Also, don't forget to mention that the `renderBottomCenterControls` are also set to display the "Paging Dots" by default too.

Reply:
@mariano-formidable Good point!